src/test/java/org/mockitousage/matchers/VarargsTest.java

```diff
@@ -363,20 +363,6 @@ public void shouldMockVarargsInvocation_multiple_vararg_matcher() {
         assertThat(mock.methodWithVarargAndNonVarargVariants("a", "b", "c")).isNull();
     }
 
-    @Test
-    public void shouldMockVarargsInvocationUsingCasts() {
```

Comment:
This test has been removed as `any()` now matches a single value passed to a varargs parameter. So this test fails with new impl. Fixing it requires changing:

```java
given(mock.methodWithVarargAndNonVarargVariants((String[]) any()))
                .willReturn("var arg method");
```

to

```java
given(mock.methodWithVarargAndNonVarargVariants(any(String[].class)))
                .willReturn("var arg method");
```

... which is no longer using casts... at this point this test becomes a duplicate of other tests such as:
* shouldMockVarargsInvocation_multiple_vararg_matcher()
* shouldDifferentiateNonVarargVariant()
* shouldMockVarargsInvocation_single_vararg_matcher()

Reply:
Hm, breaking `any()` sounds a bit scary. Is it possible to write code that works both before and after? E.g. is it possible to use `any(String[].class)` before this change as well? Ideally these are not mutually exclusive, so our users can prepare for the breaking change up front.

Reply:
@big-andy-coates I had a question here about the `any` breakage ;)